static/usage/toast/presenting/controller/vue.md

```diff
@@ -0,0 +1,26 @@
+```html
+<template>
+  <ion-button expand="block" @click="() => presentToast('top')">Present Toast At the Top</ion-button>
+  <ion-button expand="block" @click="() => presentToast('middle')">Present Toast At the Middle</ion-button>
+  <ion-button expand="block" @click="() => presentToast('bottom')">Present Toast At the Bottom</ion-button>
```

Comment:
Our previous Vue code samples have not used arrow functions, so it would be good to stay consistent.

```suggestion
  <ion-button expand="block" @click="presentToast('top')">Present Toast At the Top</ion-button>
  <ion-button expand="block" @click="presentToast('middle')">Present Toast At the Middle</ion-button>
  <ion-button expand="block" @click="presentToast('bottom')">Present Toast At the Bottom</ion-button>
```